### DISCUSSION TITLE
how to fix this error

### DISCUSSION THREAD

**Discussion by rakib91221** [runew0lf/RuinedFooocus#161] [Q&A]
how to fix this error
![11](https://github.com/user-attachments/assets/25a21541-0ffc-4be3-b6a2-15ec54b50ee2)

**Comment by yownas** [comment]
It looks like RF can't find any checkpoints/models. It might be your `settings\settings.json` that is corrupt, or if you (re)moved your models folder.

**Reply by rakib91221** [reply]
> It looks like RF can't find any checkpoints/models. It might be your `settings\settings.json` that is corrupt, or if you (re)moved your models folder.

thanks already fixed
